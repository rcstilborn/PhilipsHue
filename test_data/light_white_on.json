{
    "state": {
        "on": "true",
        "bri": "254",
        "alert": "select",
        "reachable": "true"
    },
    "swupdate": {
        "state": "noupdates",
        "lastinstall": "null"
    },
    "type": "Dimmable light",
    "name": "Hue white lamp 1",
    "modelid": "LWB014",
    "manufacturername": "Philips",
    "uniqueid": "00:17:88:01:02:94:3f:85-0b",
    "swversion": "1.15.2_r19181",
    "swconfigid": "D1D2055F",
    "productid": "Philips-LWB014-1-A19DLv3"
}

