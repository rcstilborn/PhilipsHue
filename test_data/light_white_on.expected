  {
    "name":"Hue white lamp 1",
    "id":"1",
    "on":true,
    "brightness":254
  }